storage/local/interface.go

```diff
@@ -73,12 +73,10 @@ type Querier interface {
 	// have no samples in the specified interval from the returned map. In
 	// doubt, specify model.Earliest for from and model.Latest for through.
 	MetricsForLabelMatchers(from, through model.Time, matcherSets ...metric.LabelMatchers) ([]metric.Metric, error)
-	// LastSampleForFingerprint returns the last sample that has been
-	// ingested for the given sets of label matchers. If this instance of the
-	// Storage has never ingested a sample for the provided fingerprint (or
-	// the last ingestion is so long ago that the series has been archived),
-	// ZeroSample is returned. The label matching behavior is the same as in
-	// MetricsForLabelMatchers.
+	// LastSampleForLabelMatchers returns the last sample that has been
+	// ingested for the given sets of label matchers. The label matching
```

Comment:
It may return many samples, right? One per metcher set, I guess, so perhaps:

... ingested for each of the given sets of label matchers.